lstchain/io/io.py

```diff
@@ -1131,3 +1133,49 @@ def extract_simulation_nsb(filename):
                     print('Unexpected end of %s,\n caught exception %s', filename, e)
     return nsb
 
+def check_mc_type(filename):
+    """
+    Check MC type ('point_like', 'diffuse', 'ring_wobble') using DL1/DL2 MC file
+    Parameters
+    ----------
+    filename: path
+    Returns
+    -------
+    string
+    """
+
+    simu_info = read_simu_info_merged_hdf5(filename)
+    
+    min_viewcone = simu_info.min_viewcone_radius.value
+    max_viewcone = simu_info.max_viewcone_radius.value
+
+    if max_viewcone == 0.0:
+        mc_type = 'point_like'
+
+    elif min_viewcone == 0.0:
+        mc_type = 'diffuse'
+        
+    else:
+        # check if mc_type is 'ring_wobble' or not
```

Comment:
Why do you need to check the file here? Why not just check if `min_viewcone == max_viewcone != 0`?